lib/src/value/calculation.dart

```diff
@@ -287,6 +764,19 @@ class SassCalculation extends Value {
         "${pluralize('was', args.length, plural: 'were')} passed.");
   }
 
+  /// Returns a [Callable] named [name] that calls a single argument
+  /// math function.
+  static Value _singleArgument(String name, Object argument,
+      SassNumber mathFunc(SassNumber value), bool acceptsKnownUnits) {
```

Comment:
Style nit: boolean arguments should almost always be named. Otherwise it's very unclear what they indicate at the call site.

This should also be documented. Also, "accepts known units" is a bit misleading—all functions accept known units. "requireKnownUnits" would be more accurate, since if this is true `mathFunc()` is only invoked with known units.